homeassistant/components/honeywell/__init__.py

```diff
@@ -43,15 +43,30 @@ async def async_setup_entry(hass, config):
         _LOGGER.debug("No devices found")
         return False
 
-    data = HoneywellData(hass, client, username, password, devices)
+    data = HoneywellData(hass, config, client, username, password, devices)
     await data.async_update()
     hass.data.setdefault(DOMAIN, {})
     hass.data[DOMAIN][config.entry_id] = data
     hass.config_entries.async_setup_platforms(config, PLATFORMS)
 
+    config.async_on_unload(config.add_update_listener(update_listener))
+
     return True
 
 
+async def update_listener(hass, config) -> None:
+    """Update listener."""
```

Comment:
This probably isn't needed yet, but since you're going to be changing the options flow in the other PR, it doesn't hurt at it now￼

Reply:
Yeah, there's no options flow in this integration yet.